### ISSUE TITLE
Error parsing the boinc_scheduler tag from master files

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#446]
**Reported by Thyme Lawn on 14 Jun 37866588 02:13 UTC**
There's a fault in the parsing of the "boinc_scheduler" syntax from project master files.  It's been bugging me for ages that BOINC has to repeat every scheduler request to APS@home (the first request always gets an "HTTP file not found error").

I finally got round to investigating with Wireshark and found that the there is a difference in the POST lines.  The first attempt is always 
```
POST /APS_cgi/cgi"> HTTP/1.1
```
which always gets a 404 response.  The second request is always 
```
POST /APS_cgi/cgi HTTP/1.1
```

Investigation of the master file revealed that the error must be from the way the 
```
<link rel="boinc_scheduler"href="http://www.apsathome.org/APS_cgi/cgi">
```
tag is being read, and here's the fix.
```
scheduler_op.C
309c309
<                 char* q2 = strchr(buf2, '\"');
---
>                 char* q2 = strchr(buf2, '"');
```

**Comment by romw** [comment]
**Commented by Nicolas on 15 Apr 37866601 14:13 UTC**
At last somebody found the cause! I was getting my server log full of those ```cgi">``` entries.

**Comment by romw** [comment]
**Commented by davea on 19 Jun 37867126 17:20 UTC**
(In [14167]) - client: fix bug in parsing of <link rel=boinc_scheduler ...>
    element in master files.  fixes #467

**Comment by romw** [comment]
**Commented by romw on 6 Sep 37869218 09:20 UTC**
(In [14179])     - client: fix bug in parsing of <link rel=boinc_scheduler ...>
        element in master files.  fixes #467

    client/
        scheduler_op.C
